src/panel.css

```
@@ -437,4 +437,4 @@ h3.sites-allowed {
 
 #no-trackers .fw-bottom-btn:last-of-type {
   display: none;
-}
+}
```

nit: Add extra line at end of file
```suggestion

}
```